index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Untitled-1.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <header id="header">
+        <img id="header-img" src="./images/coffe.png" width="50" height="50" alt="Coffe logo">
+        <nav id="nav-bar">
+            <ul class="nav_link">
+                <li> <a href="#">About</a></li>
+                <li><a href="#">Services</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section>
```

Comment:
- [ ] Kindly, add a class to the section. That would help you specify the only specific HTML elements you want to design in the CSS file.